### DISCUSSION TITLE
Possible malware

### DISCUSSION THREAD

**Discussion by Punkinbear4234** [community/community#129737] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I know definitely my phone is hacked. I need someone to help me with this. It's been going on way too long

**Comment by ebndev** [comment]
Hi @Punkinbear4234, thanks for being a part of the GitHub Community!

If you're looking for help for this, you might want to try asking for help somewhere that focuses on phone recovery or malware support. GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
